src/OrleansServiceBus/Providers/Streams/EventHub/EventHubBatchContainer.cs

```diff
@@ -116,5 +125,28 @@ internal static EventData ToEventData<T>(Guid streamGuid, String streamNamespace
             }
             return eventData;
         }
+
+        [SerializerMethod]
+        private static void Serialize(object obj, ISerializationContext context, Type expected)
+        {
+            var input = (EventHubBatchContainer) obj;
+            SerializationManager.SerializeInner(input.eventHubMessage, context, typeof(EventHubMessage));
+        }
```

Comment:
This does do that: SerializeInner method is currently static - it takes a context